### DISCUSSION TITLE
vertex deduplication

### DISCUSSION THREAD

**Discussion by WdsPjm** [zeux/meshoptimizer#648] [Ideas]
I have a question I would like to inquire about. For example, consider a cube. Typically, it only needs 8 vertices and 24 indices to be sent to the GPU for rendering. If mesh optimization is applied, such as vertex deduplication, would it potentially result in better performance?

**Comment by zeux** [comment]
Unsure what the question is about - isn't a cube with 8 vertices already perfectly deduplicated? fwiw typically a cube would have 24 vertices and 36 indices: 4 vertices per face because each corner has to have 3 copies to have 3 different normals, 6 indices per face because each face is two triangles.
